contractcourt/anchor_resolver.go

```diff
@@ -84,7 +84,7 @@ func (c *anchorResolver) ResolverKey() []byte {
 }
 
 // Resolve offers the anchor output to the sweeper and waits for it to be swept.
-func (c *anchorResolver) Resolve(_ bool) (ContractResolver, error) {
+func (c *anchorResolver) Resolve() (ContractResolver, error) {
```

Comment:
Hmm but according to the description this does not mean that the sweeper will immediately sweep it now but only register it ? So I don't immediatly see how the blockbeat resolves this issue ?

Reply:
> Hmm but according to the description this does not mean that the sweeper will immediately sweep it now but only register it ? 

Does or doesn't? Not sure I follow - but the sweeper only registers it, and sweeps it when the beat is received. And because we now start the consumers with a beat, it means if there's a matured input that should be swept, it will be done immediately.